### PR TITLE
(doc) Fix broken documentation link.

### PR DESCRIPTION
**Pull request by gguillotte**

### CODE REVIEW

File: documentation/config_file_master.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ canonical: "/puppetserver/latest/config_file_master.html"
 
 The `master.conf` file configures how Puppet Server handles [deprecated][] authorization methods for master endpoints. For an overview, see [Puppet Server Configuration](./configuration.html).
 
-> **Deprecation Note:** This file contains only the `allow-header-cert-info` parameter, and is deprecated as of Puppet Server 2.2 in favor of authorization settings that are configured in the [`auth.conf`][] file. Because this setting is deprecated, a default `master.conf` file is no longer included in the Puppet Server package.
+> **Deprecation Note:** This file contains only the `allow-header-cert-info` parameter, and is deprecated as of Puppet Server 2.2 in favor of authorization settings that are configured in the [new `auth.conf`][] file. Because this setting is deprecated, a default `master.conf` file is no longer included in the Puppet Server package.
```
Comment (justinstoller) [inline documentation/config_file_master.md]: I'm unclear how this fixes the link?

Comment (gguillotte) [inline documentation/config_file_master.md]: There's a reference link at the top of the document ([L8](https://github.com/puppetlabs/puppetserver/pull/1024/files#diff-171702600fa30a3e26f9c428ec7580d5L8)):

```
[new `auth.conf`]: ./config_file_auth.html
```

By changing the link text to match the reference, the link will work.

You can see in the [branch's file view](https://github.com/gguillotte/puppet-server/blob/fix-typo/documentation/config_file_master.md) that this fixes the link.

Comment (justinstoller) [inline documentation/config_file_master.md]: Ahh, I see my issue. The link is to a page that ends with ".html" (which is correct for the docs site) while in github the page is ".md". I was missing the translation that this goes through before it hits the docs site (I went to the branch's file view, and it was a valid link to a 404).

**Comment by justinstoller** [comment]
👍

**Pull request merged**